docs/content/docs.md

```diff
@@ -800,6 +800,18 @@ section](#logging) for more info on log levels.
 When this is specified, rclone condenses the stats into a single line
 showing the most important stats only.
 
+### --stats-one-line-date ###
+
+When this is specified, rclone enables the single-line stats and prepends
+the display with a date string. The default is `2006/01/02 15:04:05 - `
+
+### --stats-one-line-date-format ###
```

Comment:
I'm not convinced that we need an option for this, or maybe the option should be more powerful and allow you to set the entire format of the stats line - what do you think?

Reply:
I didn't want a separate option either. What I wanted was a way to make `--stats-one-line-date` both a `bool` and a `string` option. If the option by itself is given, it uses the default. If the option has an argument, use it for the date format.

I'm open to making it an entire reformatting, though. I'd say it'd be better to split off into a different issue.

For that issue: Would re-using the go date format logic work?

`Mon Jan 2 15:04:05 MST 2006 - 1k/2k 3% 4/5 files 6 days 7:08:09 eta`